acousticbrainz/data/__init__.py

```diff
@@ -139,13 +139,15 @@ def count_lowlevel(mbid):
 
 
 def get_summary_data(mbid):
+    """Fetches the lowlevel and highlevel features from abz database for the specified MBID."""
+    summary = {'lowlevel': None, 'highlevel': None, 'genres': None, 'moods': None, 'other': None}
     mbid = str(mbid)
     conn = psycopg2.connect(current_app.config['PG_CONNECT'])
     cur = conn.cursor()
     try:
         cur.execute("SELECT data FROM lowlevel WHERE mbid = %s", (mbid, ))
         if not cur.rowcount:
-            raise NotFound("We do not have data for this track. Please upload it!")
+            return summary
```

Comment:
I like how summary is moved into a dictionary, but this can still be an exception (not the flask one any more). In views/data.py you can catch it and set it to something else